app/static/uppy-test.html

```diff
@@ -21,7 +21,8 @@
         .use(Uppy.Tus, {
 		endpoint: '/api/v1/submissions/tus',
 		headers: {
-			'x-houston-submission-id': subId
+			//'x-houston-submission-id': subId
```

Comment:
ever going to be needed? remove this line?

Reply:
this entire .html page is a bit of a one-off just for advanced/dev use only.   i should probably either fix it up nice.... or just delete the whole thing!